app/services/beacon.ts

```diff
@@ -0,0 +1,36 @@
+import Service, { service } from '@ember/service';
+import type RouterService from '@ember/routing/router-service';
+import type RouteInfo from '@ember/routing/route-info';
+import RouteInfoMetadata, { HelpscoutBeaconVisibility } from 'codecrafters-frontend/utils/route-info-metadata';
+
+export default class BeaconService extends Service {
```

Comment:
```suggestion
export default class HelpscoutBeaconService extends Service {
```

(Might not be clear what "Beacon" means)